packages/shared/components/CollapsibleBlock.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="typescript">
+    import { Text } from 'shared/components'
+
+    export let label: string = ''
+    export let showBlock = false
+
+    const toggleBlockContent = () => {
+        showBlock = !showBlock
+    }
+</script>
+
+<style>
```

Comment:
We should remove unused / empty code; custom classes `grow` and `flex-none` are used in lines 18 / 19 but no style definitions exist.

Reply:
Those classes are tailwind, happy to remove the style block